tests/integration/it_utils.py

```diff
@@ -148,20 +150,20 @@ def send_timed_reliable_submission(
 ) -> Response:
     """
     Allows integration tests for sync clients to utilize reliable submission on
-    standalone rippled node. A one second timer to close ledger is initialized before
+    standalone rippled node. A timer to close ledger is initialized before
     running `send_reliable_submission_sync` to ensure the ledger closes and the
     submission process finishes.
 
     Arguments:
         transaction: the signed transaction to submit to the ledger. Requires a
             `last_ledger_sequence` param.
-        client: the network client used to submit the transaction to a rippled node.
+        use_json_client: boolean for choosing json or websocket client.
 
     Returns:
         The response from the server, as a Response object.
     """
     client = _choose_client(use_json_client)
-    SyncTestTimer(1, client.request, (LEDGER_ACCEPT_REQUEST,)).start()
+    SyncTestTimer(LEDGER_ACCEPT_TIME, client.request, (LEDGER_ACCEPT_REQUEST,)).start()
```

Comment:
I recommend writing a wrapper class around `threading.Timer`, so the interface is the same for `AsyncTestTimer` and `SyncTestTimer`